hw/top_earlgrey/data/ip/chip_flash_ctrl_testplan.hjson

```diff
@@ -23,7 +23,7 @@
       features: ["FLASH_CTRL.INIT.SCRAMBLING_KEYS", "FLASH_CTRL.INIT.ROOT_SEEDS",
                  "FLASH_CTRL.INFO.CREATOR_PARTITION", "FLASH_CTRL.INFO.OWNER_PARTITION"]
       stage: V2
-      si_stage: NA
+      si_stage: SV3
```

Comment:
The //sw/device/tests:flash_ctrl_test does not cover this testpoint: it only tests info pages 5 and 9, and a data page. The tests are valid and useful but have nothing to do with this testpoint, rather it tests page attributes like scrambling and access permissions. The scrambling access is weak, since write followed by read should always result in identical data regardless of scrambling, so this only catches weird errors. Perhaps a better test is to write with some setting of scrambling and read with the opposite setting: this should always result in a fatal read error.

Reply:
Kept SV3